k8s/elastic/values.yaml

```diff
@@ -149,10 +180,10 @@ kibana:
   enabled: false
   resources:
     requests:
-      cpu: "500m"
+      cpu: "512m"
```

Comment:
it's CPU, so `500m` means 1/2 CPU, and 512 won't help much :)

Reply:
ok